docs/conf.py

```diff
@@ -0,0 +1,59 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+import subprocess
+import sys
+from pathlib import Path
+
+# Assuming conf.py is located in /Users/sergeypetrakov/Documents/Documents/Skoltech_PhD/rtd_to_main/lm-polygraph/src/lm_polygraph/docs
+
+# Get the absolute path of the root directory
+root_path = Path(__file__).resolve().parents[2]
+
+# Add the src/lm_polygraph directory to the Python path
+sys.path.insert(0, str(root_path / "src" / "lm_polygraph"))
+sys.path.insert(0, str(root_path))
+
+# Install the library using pip
+subprocess.run(["pip", "install", "-e", "."], cwd=root_path)
+
+
+project = "LM-Polygraph"
+copyright = "2023, MBZUAI"
+author = "MBZUAI"
+release = "0.3.0"
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+extensions = [
+    "sphinx.ext.todo",
+    "sphinx.ext.viewcode",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.duration",
+    "sphinx.ext.doctest",
+    "sphinx.ext.autosummary",
+]
+
+templates_path = ["_templates"]
+exclude_patterns = ["Thumbs.db", ".DS_Store"]  # '_build',
+# autodoc_mock_imports = ['lm_polygraph']
```

Comment:
What are these comments? Do we need them?

Reply:
droped